### DISCUSSION TITLE
Error with Android 32 and Skia 6.2.0

### DISCUSSION THREAD

**Discussion by samuelevigilia** [skia4delphi/skia4delphi#337] [General]
I have installed Skia4Delphi V6.2.0 but when I try to build my FMX app on the Android 32 platform I get this error : 
arm-linux-androideabi-ld.exe error: cannot find -lsk4d .
With Skia4Delphi V6.1.0 and V6.0.0 I don't get this error so I installed V6.1.0 because I Update my IDE to Athens 12.1

**Comment by viniciusfbb** [comment]
@samuelevigilia Can you try to delete the output folder of your project with the IDE closed?

**Reply by viniciusfbb** [reply]
@samuelevigilia Please, give the feedback, otherwise we will close this discussion.
